mods/cnc/audio/music.yaml

```diff
@@ -66,3 +66,7 @@ nod_map1: Nod Map Theme
 	Hidden: true
 nod_win1: Nod Win Theme
 	Hidden: true
+
+# Special effects
```

Comment:
#9488 lets you move these into the map file itself, so adding [has dependencies] for that and #9666.